lib/archive.mli

```diff
@@ -35,9 +35,6 @@ val bzip2 :
   dry_run:bool -> ?force:bool -> dst:Fpath.t -> string -> (unit, R.msg) result
 (** [bzip2 dst s] compresses [s] to [dst] using bzip2. *)
 
-val ensure_tar : unit -> (unit, R.msg) result
```

Comment:
Hmm I'd be curious to know why this is not used anymore. As long as we have a decent error message if `tar`'s missing I'm fine with it!

Reply:
`tar` is only used in `Archive.untbz` that already checks that tar is present itself